src/DIRAC/WorkloadManagementSystem/Service/JobStateUpdateHandler.py

```diff
@@ -57,14 +57,12 @@ def initializeHandler(cls, svcInfoDict):
     @classmethod
     def export_updateJobFromStager(cls, jobID, status):
         """Simple call back method to be used by the stager."""
-        if status == JobStatus.DONE:
+        if status == "Done":
```

Comment:
Why not
```suggestion
        if status == JobStatus.DONE:
```

Reply:
Because this is not the same "Done": what I mean is that it's "by chance" that `JobStatus.DONE == "Done"`.